integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiJmsRecovery.java

```diff
@@ -0,0 +1,739 @@
+                         + dbNodePort + "/devpdb.k8s";
+    logger.info("ConnectionPool URL = {0}", cpUrl);
+    assertDoesNotThrow(() -> createDatabaseSecret(dbSecretName, 
+            "sys as sysdba", "Oradoc_db1", cpUrl, domainNamespace),
+            String.format("createSecret failed for %s", dbSecretName));
+    String configMapName = "jdbc-jms-recovery-configmap";
+
+    createConfigMapAndVerify(
+        configMapName, domainUid, domainNamespace,
+        Arrays.asList(MODEL_DIR + "/jms.recovery.yaml"));
+
+    // this secret is used only for non-kind cluster
+    createSecretForBaseImages(domainNamespace);
+
+    // create PV, PVC for logs/data
+    createPV(pvName, domainUid, ItMiiJmsRecovery.class.getSimpleName());
+    createPVC(pvName, pvcName, domainUid, domainNamespace);
+
+    // create job to change permissions on PV hostPath
+    createJobToChangePermissionsOnPvHostPath(pvName, pvcName, domainNamespace);
+
+    // create the domain CR with a pre-defined configmap
+    createDomainResource(domainUid, domainNamespace, adminSecretName,
+        OCIR_SECRET_NAME, encryptionSecretName,
+        replicaCount, configMapName, dbSecretName);
+
+    // wait for the domain to exist
+    logger.info("Check for domain custom resource in namespace {0}", domainNamespace);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for domain {0} to be created in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                domainUid,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(domainExists(domainUid, DOMAIN_VERSION, domainNamespace));
+
+    logger.info("Check admin service and pod {0} is created in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+    // create the required leasing table 'ACTIVE' before we start the cluster
+    createLeasingTable();
+    // check managed server services and pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server services and pods are created in namespace {0}",
+          domainNamespace);
+      checkPodReadyAndServiceExists(managedServerPrefix + i, domainUid, domainNamespace);
+    }
+  }
+
+  /**
+   * Verify all server pods are running.
+   * Verify all k8s services for all servers are created.
+   */
+  @BeforeEach
+  public void beforeEach() {
+    logger.info("Running beforeEach() ....");
+  }
+
+  /**
+   * Verify JMS Service is migrated to an available active server.
+   * Here the JMS messages are stored in File store on PV
+   */
+  @Test
+  @Order(1)
+  @DisplayName("Verify JMS Service migration with FileStore")
+  public void testMiiJmsServiceMigrationWithFileStore() {
+
+    buildClientOnPod();
+    runJmsClientOnAdminPod("send", 
```

Comment:
see if you can call CommonTestUtils.runClientInsidePod.

Reply:
done